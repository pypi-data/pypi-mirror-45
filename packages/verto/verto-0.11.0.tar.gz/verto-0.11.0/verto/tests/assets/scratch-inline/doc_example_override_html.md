How about some scratch: `scratch:when flag clicked`
