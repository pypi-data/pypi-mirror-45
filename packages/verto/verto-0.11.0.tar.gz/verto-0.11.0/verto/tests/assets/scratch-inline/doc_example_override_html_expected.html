<p>How about some scratch: <img class="scratch-blocks" onerror="this.src='scratch-blocks-2f3ea223b778227287b8935bc5d209e25d3e8a25ef46ff85f6c44818159601d7.png'" src="scratch-blocks-2f3ea223b778227287b8935bc5d209e25d3e8a25ef46ff85f6c44818159601d7.svg" /></p>
